qiime2/sdk/plugin_manager.py

```diff
@@ -142,34 +144,75 @@ def get_semantic_types(self):
 
         return types
 
+    class FormatFilters(enum.Flag):
```

Comment:
These enum values will need to be accessible for SDK developers to import and actually _use_ them (see my comments above about the Artifact API, similar story).